### PR TITLE
Changes to install.sh

### PR DESCRIPTION
**Pull request by nanospearing**
- Added a --install-terraform argument
- Added a unzip command check to the standalone function
	- Cleaner error and help redirect the user to a solution
- Added help info for `--install-terraform` argument
- Fixed standalone install typo (ard64 -> arm64)

### CODE REVIEW

**Comment by KozmikNano** [comment]
I have read the CLA Document and I hereby sign the CLA

File: install.sh [hunk 1]
```diff
@@ -54,6 +54,15 @@ Usage:
   --rsh <bin>
       Specifies the remote shell for remote installation. Defaults to ssh.
 
+  --install-terraform
```
Comment (kylecarbs) [inline install.sh:57]: Since this script is called `install.sh`, what do you think about calling this `--with-terraform`?

Comment (kylecarbs) [reply install.sh:57]: Or since it stops early and just installs Terraform, what about just `--terraform`?

File: install.sh [hunk 2]
```diff
@@ -351,6 +369,43 @@ fetch() {
...
+install_terraform() {
+	# Check if the unzip package is installed. If not error peacefully.
+	if !(command_exists unzip); then
+		echoh
+		echoerr "This script needs the unzip package to run."
+		echoerr "Please install unzip to use this function"
+		exit 1
+	fi
+
+	echoh "Installing Terraform version 1.3.4 $TERRA_ARCH package from Hashicorp Source."
```
Comment (kylecarbs) [inline install.sh:381]: I think we should store the Terraform version in an environment variable above.

File: install.sh [hunk 3]
```diff
@@ -251,12 +263,14 @@ main() {
...
 	CACHE_DIR=$(echo_cache_dir)
 	STANDALONE_INSTALL_PREFIX=${STANDALONE_INSTALL_PREFIX:-/usr/local}
+	TERRAFORM_INSTALL_PREFIX=${TERRAFORM_INSTALL_PREFIX:-/usr/local}
 	STANDALONE_BINARY_NAME=${STANDALONE_BINARY_NAME:-coder}
 	VERSION=${VERSION:-$(echo_latest_version)}
 	# These can be overridden for testing but shouldn't normally be used as it can
 	# result in a broken coder.
 	OS=${OS:-$(os)}
 	ARCH=${ARCH:-$(arch)}
+	TERRA_ARCH=${TERRA_ARCH:-$(terra_arch)}
```
Comment (kylecarbs) [inline install.sh:273]: Since `TERRAFORM_` has been used above, I think for consistency we should have them all be `TERRAFORM_`.

**Pull request closed without merging**